### ISSUE TITLE
size of the graph is not as set

### ISSUE THREAD

**Issue by sharahamo** [kupiqu/plot2svg#5]
When the figure is saved to svg, the size of the figure is not as what I have initially set. It sorts of becomes adjusted to a preset size (see figure). 

Please advice on the workaround for this. 
<img width="728" alt="screen shot 2018-06-28 at 09 29 35" src="https://user-images.githubusercontent.com/8395313/42022706-e2339b0c-7ab5-11e8-8230-483897272d85.png">
<img width="1052" alt="screen shot 2018-06-28 at 09 29 41" src="https://user-images.githubusercontent.com/8395313/42022707-e24dcae0-7ab5-11e8-8c0b-a77d1c72e589.png">

**Comment by kupiqu** [comment]
I assume you took this into account:

```
  %  Usage: plot2svg(filename, id, debug, legendIcons, clippingMode, figureSize, pixelFileType, creditPrintBool)
  %
  %         all arguments are optional
  %
  %         1. filename: name of the svg file (including extension)
  %         can be omitted (nargin < 1) or left empty ('' or []) and a system dialog will be prompted
  %
  %         2. id: graphic handle (if omitted or empty, current gcf is taken)
  %
  %         3. debug = 0 or 1 (enable to ease debugging of plot2svg functionality; 0 if omitted or empty)
  %
  %         4. legendIcons: legend pieces (necessary in new matlab versions to respect legend appearance)
  %         can be omitted (nargin < 4) or left empty ('' or [])
  %
  %         5. clippingMode = 0 (no clipping) or 1 (box on/off dependent clipping) or 2 (strict axes clipping) or 3 (axes+data dependent clipping)
  %         clippingMode is set to 1 if argument is omitted or empty
  %
  %         6. figureSize = [width,height] (actual figure size if omitted or empty)
  %
  %         7. pixelFileType = 'png' or 'jpg' (png used if omitted or empty)
  %
  %         8. creditPrintBool - logical argument for whether to display credit information (1 if omitted or empty)
```

If so, please add a self-contained code snippet I can take a look at

**Comment by sharahamo** [comment]
I basically just the plot2svg('filename.svg') -- will this automatically take the current figure size or not?From my understanding it will take the figure size automatically.

**Comment by kupiqu** [comment]
>  ... what I have initially set.

this seemed to me as you set the size in plot2svg, but now I understand you didn't and this was the size of the figure in matlab itself

Well, it should maintain the size (although scaled according to dpi), but anyway it seems you found an issue I didn't encounter. Unless you share a failing self-contained case with me, I don't know how do you expect me to reproduce and fix this... Also, if willing to help, the minimal the code snippet able to show this issue, the better...

**Comment by kupiqu** [comment]
Mmm, I'll check if this happens in a vanilla subplot figure... It's all I can imagine right now being wrong according to your screenshots

**Comment by sharahamo** [comment]
basically with any plot, I will do this: 

`set(gcf, 'Position', [209,350,1072,348])`

but it won't take the size I have provided above. If i just use `plot2svg`, it won't work but if I use `plot2svg('test.svg', [], [], [],1, [1072,348])` it works fine. 

This is not a big issue anyhow.

**Comment by kupiqu** [comment]
ok, I'll check it out...

**Comment by kupiqu** [comment]
Closing. Fixed the issue in commit a596f00fc039446d5be5a51ac62e439a73b7a197

**Comment by kupiqu** [comment]
Not sure you're interested in knowing the reason causing this bug, but anyway, this happenned because my plot2svg version builds a temporal copy of the original figure to avoid any disruption of the original figure. Apparently the size is not copied automatically when copying the figure content, so I had to set the figure size manually. 

Weird that I didn't find this issue before. I guess because I don't frequently use figure proportions very different than default...
